src/main/resources/cli_en_US.properties

```diff
@@ -0,0 +1,56 @@
+#
+# This file is part of the DITA Open Toolkit project.
+#
+# Copyright 2020 Jarno Elovirta
+#
+# See the accompanying LICENSE file for applicable license.
+#
+error_msg=Error: %s
+help.option.help=Print help information
+help.option.debug=Enable debug logging
+help.option.verbose=Enable verbose logging
+# Version subcommand
+version=DITA-OT version %s
+version.usage=dita version [options]
+# Plugins subcomand
+plugins.usage=dita plugins [options]
+# Deliverables subcommand
+deliverables.usage=dita deliverables <file> [options]
+deliverables.argument.file=Project file
+deliverables.error.project_not_defined=Project file not defined
+# Install subcommand
+install.usage=dita install [<id> | <url> | <file>]
+install.argument.id=Install specified plug-in ID from registry
+install.argument.url=Install plug-in from a URL
+install.argument.file=Install plug-in from a local ZIP file
+install.option.force=Force-install plug-in (overwrite existing version)
+# Uninstall subcommand
+uninstall.usage=dita uninstall <id>
+uninstall.argument.id=Uninstall plug-in with the specified ID
+uninstall.error.identifier_not_defined=You must specify plug-in identifier when using the uninstall subcommand
+# Transtypes subcommand
+transtypes.usage=dita transtypes [options]
+# Conversion command
+conversion.usage.input=dita deliverables <file> [options]
```

Comment:
Because we can't have an array of strings for usage, I used named usage strings.